### ISSUE TITLE
In eager mode, how to achieve weight attenuation during training, and how to customize a layer learning rate？

### ISSUE THREAD

**Issue by TCBocean** [tensorflow/tensorflow#33500]
My environment:
Tensorflow1.14-gpu
Hello, I have set up a CNN model in eager mode. Now, I want to set a weight decay, which I did on the old version:
`       
reg_losses = tf.get_collection(tf.GraphKeys.REGULARIZATION_LOSSES)

print("Regularization losses:")

opt_loss = loss + args.weight_decay*sum(reg_losses) 
 `
But in eager mode, this method is not very suitable. Is there any new way to provide it? I did not find a tutorial.

In addition, I want to customize the learning rate of some layers. For example, the feature extraction part of the network (such as mobilenetv3) only needs to use a small learning rate, while other parts need a large learning rate. How to achieve this?

I hope someone can help me, thank you very much.

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.
 Thanks!
